sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -70,13 +70,13 @@
       cross platform compatibility within SDKs -->
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-storage</artifactId>
-      <version>8.3.0</version>
+      <version>8.4.0</version>
```

Comment:
@gapra-msft These are the versions we'd discussed.

Reply:
Looks good to me